### PR TITLE
ADCM-2062 Tests on presence of "config" section in action bundle description

### PR DESCRIPTION
**Pull request by Sealwing**

### CODE REVIEW

File: tests/functional/test_action_config.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    """
+    Action's configuration not presented
+    Run action with config
+    """
+    expected_message = (
+        "The task includes an option with an undefined variable. "
+        "The error was: 'dict object' has no attribute 'config'"
+    )
+    with allure.step('Run action without config and expect it to fail with ansible error'):
+        task = run_cluster_action_and_assert_result(cluster, 'no_config', 'failed', config={'some_param': 1})
```
Comment (skhomuti) [inline tests/functional/test_action_config.py:43]: it should be 400 error on validating config

**Comment by skhomuti** [comment]
Also, I suggest check actions on other cluster and provider objects too

File: tests/functional/test_action_config.py (hunk 1 above)
Comment (Sealwing) [reply tests/functional/test_action_config.py]: Changed and added cluster/provider related object's actions

File: tests/functional/test_action_config.py [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+                    ACTION_MAP[obj.__class__](obj, 'with_config', 'failed')
+                except ErrorMessage as e:
+                    _check_error_message(e, 400)
+                else:
+                    raise AssertionError("Action run should've failed")
+            with allure.step("Check that job wasn't launched"):
+                _check_job_count_equal_to(job_count, sdk_client_fs)
+
+
+def _check_error_message(error: ErrorMessage, expected_code: int):
```
Comment (skhomuti) [inline tests/functional/test_action_config.py:124]: look at CONFIG_KEY_ERROR from `errorcodes` module. It will be enough

Comment (Sealwing) [reply tests/functional/test_action_config.py:124]: ty

**Comment by Sealwing** [comment]
https://allure-ee.adsw.io/launch/14134/tree/5919644?search=W3siaWQiOiJzdGF0dXMiLCJ0eXBlIjoidGVzdFN0YXR1c0FycmF5IiwidmFsdWUiOlsiZmFpbGVkIl19XQ%3D%3D&treeId=7

**Review by skhomuti — APPROVED** [review]

**Review by vgorkavenko — APPROVED** [review]

**Pull request merged**